docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -695,6 +695,12 @@ UDP
   If the original connection was redirected by iptables TPROXY, and the listener's transparent
   option was set to true, this represents the original destination address and port.
 
+  .. note::
```

Comment:
This code block is copied from `DOWNSTREAM_DIRECT_REMOTE_ADDRESS` just to be consistent.